src/buyingcatalogue/values.yaml

```diff
@@ -611,4 +611,37 @@ redis-commander:
   useLocalImage: false
   redisHost:
     name: "bc-buyingcatalogue"
-    key: redis-url
\ No newline at end of file
+    key: redis-url
+
+pb-ac-tests:
+  enabled: true
+  replicaCount: 1
+  image:
+    repository: gpitfuturesdevacr.azurecr.io/nhsd/buying-catalogue/pb-ac-tests
+  useLocalImage: false
+  imagePullSecrets: 
+  nameOverride: "pb-ac-tests"
+  fullnameOverride: "gpitfutures-bc-pb-ac-tests"
+  hubUrl: "http://sel-grid-selenium-hub.selenium-grid:4444/wd/hub"
```

Comment:
how does it know which selenium grid to use?

Reply:
We should only need a single grid per environment to run all the tests

Reply:
When installing the selenium-grid chart, it outputs the DNS name of the hub. Since the name of the namespace is static and there's always one per environment, I don't think we need to figure out the location of the hub.